cassandra-mesos-model/src/main/java/io/mesosphere/mesos/util/ProtoUtils.java

```diff
@@ -88,8 +101,9 @@ public static Resource ports(@NotNull final Iterable<Long> ports, @NotNull final
     }
 
     @NotNull
-    public static Resource scalarResource(@NotNull final String name, final double value, @NotNull final String role) {
-        return Resource.newBuilder()
+    public static Resource scalarResource(@NotNull final String name, final double value,
```

Comment:
If wrapping is needed please place each parameter on it's own line, see https://github.com/mesosphere/cassandra-mesos/blob/c16e97f093750039791526eeb52bd7b97a9c4c74/cassandra-mesos-model/src/main/java/io/mesosphere/mesos/util/ProtoUtils.java#L126-126.